### DISCUSSION TITLE
Vector Tiles

### DISCUSSION THREAD

**Discussion by Cdingram** [react-native-maps/react-native-maps#4912] [Feature requests]
Support for [vector tiles](https://en.wikipedia.org/wiki/Vector_tiles) would be an amazing addition to this library now that it’s becoming industry standard. I love this library but the lack of support forced me to use [rnmapbox](https://github.com/rnmapbox/maps/blob/main/docs/VectorSource.md) which I don’t like working with as much. For web, leaflet has a [great plugin](https://github.com/Leaflet/Leaflet.VectorGrid) that implements all kinds of vector tile sources.

**Comment by Shrey00** [comment]
I also required to use vector tiles, since most maps do not provide their official SDK for react native, vector tiles seems to be the best way to show maps. I am trying to use Ola maps, a great alternative for google maps, the pricing is really good, but they don't have any official SDK for react native. I am also looking to use rnmapbox or maplibre gl.
